### PR TITLE
Changing the reporter.js to use the quantity of files

### PR DESCRIPTION
**Pull request by udjamaflip**
Currently the report (when all are passed) will just show up as 1 test rather than the quantity of tests which are done, this change will always report how many files have been tested regardless of any errors.

### CODE REVIEW

**Comment by xjamundx** [comment]
Thanks for this. I'll look into this later today if possible!

**Comment by udjamaflip** [comment]
Cheers, I just made a second relevant update to make it more syntactically correct. It should now also output an empty "testcase" element for each passed AND failed test.

**Comment by xjamundx** [comment]
Great @udjamaflip  would have any interest in becoming a co-owner of the project?

File: reporter.js [hunk 1]
```diff
@@ -10,84 +10,90 @@ var wrStream;
...
-    "<": "&lt;",
-    ">": "&gt;"
-  };
-  for (var r in pairs) {
-    if (typeof(s) !== "undefined") {
-      s = s.replace(new RegExp(r, "g"), pairs[r]);
-    }
-  }
-  return s || "";
+	var pairs = {
```
Comment (xjamundx) [inline reporter.js]: Maybe we should add a linter to enforce some convention

**Comment by udjamaflip** [comment]
@xjamundx Definitely, JSHint needs this type of reporter and it's the only one out there that really works. Happy to help keep it up-to-date.

File: reporter.js (hunk 1 above)
Comment (udjamaflip) [inline reporter.js]: Sorry - I tried to match your indentation but it looks like my editor took over. Do you prefer tabs or spaces?

**Comment by udjamaflip** [comment]
Because my editor obfuscated everything, the line I changed in my original commit is here:
https://github.com/xjamundx/jshint-junit-reporter/pull/10/files#diff-fdfe8a4d956efc7b79e8d0c19bf2e0f6R67

Basically using the original files list count rather than the list of files with failures. This also means if a user targets an empty directory they'll get a testsuite with 0 files rather than 1.

Let me know if you want me to convert back to spaces indentation.

**Comment by xjamundx** [comment]
I don't mind at all. I'm going to make you owner and let you publish this yourself. What's your NPM account?

**Comment by udjamaflip** [comment]
My NPM account is "udjamaflip" - I'll change the version to 1.1 so people don't get snarled up in a change which might modify their workflow

**Comment by xjamundx** [comment]
awesome, you should now have access!

```

~/dev/jshint-junit-reporter (master) $ npm owner ls

xjamundx <jamund@gmail.com>

udjamaflip <andy@sharman.io>

```

**Pull request merged**